tests/rptest/tests/acls_test.py

```diff
@@ -56,7 +84,8 @@ def prepare_users(self):
         client.acl_create_allow_cluster("cluster_describe", "describe")
 
     @cluster(num_nodes=3)
-    def test_describe_acls(self):
+    @matrix(tls=[True, False])
+    def test_describe_acls(self, tls):
```

Comment:
`tls` seems unused.

Reply:
It's being snooped in __init__ via test_context -- I'd prefer if we didn't do that though, it makes it harder to read.

Could RedpandaService have a set_security_settings similar to set_resource_settings?  That way the tests class could defer calling RedpandaService.start() until the test itself, and at that point know whether to construct a TLS config.

Reply:
Fixed